avocado/core/main.py

```diff
@@ -19,6 +19,7 @@
 import traceback
 
 try:
+    import avocado.init
```

Comment:
This is a quick hack to get `avocado run` working.

Reply:
Yes, I got it. My comment it is because we had a few issues in the past with the order of config initialization. By doing a first look this is not breaking the settings, but I will need to test with some cases.